package/gluon-state-check/Makefile

```diff
@@ -0,0 +1,28 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-state-check
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/$(PKG_NAME)
+  TITLE:=Provides info about the routers state
+  DEPENDS:=+gluon-core +micrond
+endef
+
+define Package/$(PKG_NAME)/description
```

Comment:
Still a $(PKG_NAME) left here.

Reply:
all done in ce76c7b69d1d10ad39583ffb2dcfc5defb036ce4